node/narwhal/src/helpers/storage.rs

```diff
@@ -597,7 +597,7 @@ pub mod tests {
     type CurrentNetwork = snarkvm::prelude::Testnet3;
 
     /// Asserts that the storage matches the expected layout.
-    fn assert_storage<N: Network>(
+    pub fn assert_storage<N: Network>(
```

Comment:
it's not from this PR, but all of the params to this function should be references - there is zero need to pass them by value, which requires cloning in tests

Reply:
Yeah, good catch. I can change that in separate PR.